src/view/com/composer/text-input/web/EmojiPicker.web.tsx

```diff
@@ -58,21 +58,27 @@ export function EmojiPicker({close}: {close: () => void}) {
     // eslint-disable-next-line react-native-a11y/has-valid-accessibility-descriptors
     <TouchableWithoutFeedback onPress={close} accessibilityViewIsModal>
```

Comment:
@gaearon The ancestor is simply just this one.

I was following the pattern in the `Composer.web.tsx` file and the `Modal.web.tsx` file, but your idea sounds good. 

Let me take a look

Reply:
Either way should work

Reply:
Doesn't seem like the React Native `ref` on a `View` has a `contains` method

Reply:
probably not worth sussing out the other approach, but I _believe_ there's a way to get the dom element out of the ref in the RNW build